### ISSUE TITLE
Is this project deprecated or not maintained or both?

### ISSUE THREAD

**Issue by anthonygore** [theocodes/vue-brunch#26]
The readme says its not maintained, the description says it's deprecated.

If deprecated, what's replacing it?

Looks like a really useful plugin so I hope it stays!

**Comment by bigfang** [comment]
Hi, I fixed bugs that I has been found, repo is [here](https://github.com/bigfang/vue-brunch)

**Comment by tomquirk** [comment]
Awesome @bigfang. Would be good to have a single repo to work with though!

**Comment by bigfang** [comment]
@tomquirk you mean we‘d better keep improve this project? if @theocodes decide re-maintained this project, I will glad to send pr to.

last night(UTC+8) I have published a npm named vuejs-brunch, then I noticed your pr. It is great especially tests. so I think you have more experience on javascript than me. but I am not very good at english. If this (publish npm) is not appropriate, plz let me know.

**Comment by tomquirk** [comment]
Oh yeah, not being sarcastic! I was just making a comment that it's always better to consolidate efforts into a single project.

@theocodes would be great to get your thoughts 😄

**Comment by theocodes** [comment]
Hi all,

Sorry for the delayed reply... Truth is, I'm struggling to find the time to properly look after the project at the minute. I've recently had a baby, for now, unfortunately, my efforts will be a bit limited.

I'm happy to take PR's or even bring in someone who may be interested in helping to maintain the project.

I'm hoping I will be able to find more free time in the near future but for the time being, as I said, please get in touch if you'd like to get involved :)

Thanks and sorry for the inconvenience!

**Comment by tomquirk** [comment]
Hey mate,

Great to hear from you. Congrats on the kid!

I've just cleaned up my [PR](https://github.com/theocodes/vue-brunch/pull/25) as per @nblackburn 's comments so that should be ready to come in.

I would be happy for you to bring me on as a collaborator if you see it necessary, but I'm just as happy if you want to accept PR's yourself. I can't say I'll be able to commit 100% to general maintenance but will certainly be able to come in and help fix major bugs.

**Comment by theocodes** [comment]
@tomquirk thanks for the PR. I've merged it in. I've also gone ahead and added ci so it makes the process of reviewing PR's easier over time (potentially as we improve tests).

I've removed the deprecated message from subheader and with some help hope to start driving the project to where it needs to be. 

As I explained above, my time is quite limited now but I will try and find a couple of hours each week to go through issues, PRs etc.. 

Thanks for the help! :+1:
